src/frontend/src/components/organisations/search/__tests__/index.test.tsx

```diff
@@ -23,27 +23,29 @@ const expectParentElementToNotInclude = (htmlElement : HTMLInputElement, errorMe
   expect(htmlElement?.parentElement?.textContent?.includes(errorMessage)).toBeFalsy();
 };
 
+const mockOnSearch = jest.fn();
+
 describe('OrganisationsSearch', () => {
 
   it('should render the component without crashing', () => {
-    render(<OrganisationsSearch />);
+    render(<OrganisationsSearch onSearch={mockOnSearch} />);
```

Comment:
can we extract the setup into a beforeEach block to avoid duplication  `  beforeEach(() => {
    render(<OrganisationsSearch onSearch={mockOnSearch} />);
  });`

Reply:
done as requested